templates/tools/dockerfile/cxx_build_interop.sh.include

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Builds C++ interop server and client in a base image.
+set -e
+
+mkdir -p /var/local/git
+git clone /var/local/jenkins/grpc /var/local/git/grpc
+# clone gRPC submodules, use data from locally cloned submodules where possible
+# <%text>$</%text>{name} is required to escape mako.
+(cd /var/local/jenkins/grpc/ && git submodule foreach 'cd /var/local/git/grpc \
```

Comment:
nit: rendering the template gets rid of trailing backslashes. You'll have to use sth like ${"\\"} (see other templates) to render a backslash.

grpc_interop_cxx/build_interop.sh should have no changes.